src/Core/src/Platform/iOS/KeyboardAutoManagerScroll.cs

```diff
@@ -319,6 +319,12 @@ internal static void AdjustPosition()
 
 		if (View.FindResponder<UINavigationController>() is UINavigationController navigationController)
 		{
+			if (View.IsDescendantOfView(navigationController.NavigationBar))
```

Comment:
In the original video it seems like it's shifting the page (ContentView) in a negative direction. Would it be possible to detect that the focused view isn't a child of page and then just exit at that point. Just curious if there's a more general solution here.

Reply:
The negative shift is because the scrolling logic thinks that entry in the navbar is actually behind the navbar and needs to scroll the page down to get it below the navbar. 

Do you mean if the view is a child of anything that inherits from the Page class?

Reply:
I wonder how this would affect things from third parties potentially?